tools/webpack/vendors.js

```diff
@@ -0,0 +1,96 @@
+/**
+ * External dependencies
+ */
+const CopyWebpackPlugin = require( 'copy-webpack-plugin' );
+const { join } = require( 'path' );
+
+/**
+ * WordPress dependencies
+ */
+const DependencyExtractionWebpackPlugin = require( '@wordpress/dependency-extraction-webpack-plugin' );
+
+const copiedVendors = {
+	'inert-polyfill': [
+		'wicg-inert/dist/inert.js',
+		'wicg-inert/dist/inert.min.js',
+	],
+};
+const importedVendors = {
+	react: { import: 'react', global: 'React' },
+	'react-dom': { import: 'react-dom', global: 'ReactDOM' },
```

Comment:
Technically changing how we import react and react Dom is not mandatory for this PR but since we need this config for react-jsx-runtime, I just applied it to react and react-dom too to move away from UMD builds.